otter/models/cass.py

```diff
@@ -1215,7 +1216,17 @@ def _delete_group():
         def _delete_lock_znode(result):
             d = self.kz_client.delete(LOCK_PATH + '/' + self.uuid,
                                       recursive=True)
-            d.addCallback(lambda _: result)
+
+            def return_result_ignore_errors(zk_delete_result):
+                # If we fail to delete the node, log and otherwise ignore it.
+                if isinstance(zk_delete_result, Failure):
+                    self.log.msg(
+                        "Error cleaning up lock path (when deleting group)",
+                        exc=zk_delete_result.value)
+
+                return result
+
+            d.addBoth(return_result_ignore_errors)
```

Comment:
Yes, you're right, apologies, forgot about that.  I basically went with what you suggested anyway, but just returned None at the end, since apparently that's what the interface says is the success result anyway